src/KK.AspNetCore.EasyAuthAuthentication/AuthenticationTicketBuilder.cs

```diff
@@ -12,21 +12,20 @@ internal static class AuthenticationTicketBuilder
         /// Build a `AuthenticationTicket` from the given payload, the principal name and the provider name.
         /// </summary>
         /// <param name="claimsPayload">A array of JObjects that have a `type` and a `val` property.</param>
-        /// <param name="userid">The user ID of the current user.</param>
         /// <param name="providerName">The provider name of the current auth provider.</param>
+        /// <param name="options">The <c>EasyAuthAuthenticationOptions</c> to use.</param>
         /// <returns>A `AuthenticationTicket`.</returns>
-        public static AuthenticationTicket Build(IEnumerable<JObject> claimsPayload, string userid, string providerName)
+        public static AuthenticationTicket Build(IEnumerable<JObject> claimsPayload, string providerName, EasyAuthAuthenticationOptions options)
```

Comment:
Make it sence to put `providerName` in the `options` too?

Reply:
okay I see it. It is a little bit other context. the `options` more the programm implementation and the `providerName` is something that comes from the authentification workflow. So it is okay